scout/server/blueprints/cases/templates/cases/case.html

```diff
@@ -238,49 +238,48 @@ <h4 class="card-header">Case: {{case.display_name}}</h4>
-                    <button type="submit" class="input-group-append btn btn-outline-secondary text-secondary form-control">Label</button>
-                </div>
-              </form>
-            </div>
-            <div class="col text-right">
-              <div class="btn-group">
-                <button type="button" class="btn btn-outline-secondary form-control" data-toggle="modal" data-target="#add-case-group-{{group_id}}">
-                  <span class="fa fa-plus-square-o"></span>&nbsp;Bind other case
-                </button>
+          <form class="form-inline mt-2" action="{{ url_for('cases.case_group_update_label', case_group=group_id)}}" method="POST">
+            <div class="input-group mb-2 mr-sm-2">
+              <div class="input-group-prepend">
+                <div class="input-group-text"><i class="fa fa-users"></i></div>
               </div>
+              <input type="text" class="form-control" name="label" id="case_group_label-{{group_id}}" placeholder="{{ case_group_label[group_id] }}">
             </div>
-          </div>
-          <div class="row mb-4">
-            <div class="col-12">
-              <ul class="list-group">
-              {% for grouped_case in case_group %}
-                <li class="list-group-item">
-                    <a href="{{ url_for('cases.case', institute_id=grouped_case.owner, case_name=grouped_case.display_name) }}"">{{ grouped_case.display_name }}</a>
-                    <span class="pull-right">
-                      <a href="{{ url_for('cases.remove_case_group', institute_id=institute._id, case_name=grouped_case.display_name, case_group=group_id) }}" class="btn btn-link btn-sm">
+            <button type="submit" class="btn btn-secondary btn-sm mb-2">save label</button>
+            <button type="button" class="btn btn-secondary btn-sm mb-2 ml-3 text-right" data-toggle="modal" data-target="#add-case-group-{{group_id}}">
+              <i class="fas fa-user-plus" data-toggle='tooltip' title="Add other case to this group"></i>
+            </button>
+          </form>
+
+          <div class="col-12">
+            {% for grouped_case in case_group %}
+              <span class="badge badge-pill badge-light">
+                <a href="{{ url_for('cases.case', institute_id=grouped_case.owner, case_name=grouped_case.display_name) }}"">{{ grouped_case.display_name }}</a>
+                  <span>
+                    <a href="{{ url_for('cases.remove_case_group', institute_id=institute._id, case_name=grouped_case.display_name, case_group=group_id) }}" class="btn btn-link btn-sm">
                         <span class="fa fa-remove"></span></a>
-                    </span>
-                </li>
-              {% endfor %}
-              </ul>
-            </div>
+                  </span>
+              </span>
+            {% endfor %}
+            <hr style="height:2px;">
```

Comment:
Perhaps make that ruler join like and show it only if there is a next element (eg before all except the first)? It looks a little lonely over the card border for the last one?